lib/rbnacl/password_hash.rb

```diff
@@ -19,9 +19,9 @@ module RbNaCl
   # All of them also take a CPU work factor, which increases the amount of
   # computation needed to produce the digest.
   module PasswordHash
-    # scrypt: the original sequential memory hard password hashing function.
-    # This is also the only password hashing function supported by libsodium,
-    # but that's okay, because it's pretty awesome.
+    # scrypt
+    #
+    # deprecated in favor of argon2
```

Comment:
scrypt isn't "deprecated". It's actually still my preferred choice (I say this as a PHC judge)